mscore/instrwidget.h

```diff
@@ -16,6 +16,8 @@
 #include "ui_instrwidget.h"
 #include "libmscore/clef.h"
 
+class QTreeWidgetItem;
```

Comment:
This forward declaration isn't strictly necessary to compile due to the way MuseScore uses pre-compiled headers. However, it is necessary to compile without the use of pre-compiled headers. Is there a policy on whether these should/shouldn't be included?

Reply:
It doesn't influence the compilation speed, so let's keep it.

Reply:
Good. It stops IDEs from complaining about known types, that that should benefit everyone.